### PR TITLE
Solved issue #522 

### PR DESCRIPTION
**Pull request by aliya-rahmani**
Simplified language in translation page

### CODE REVIEW

File: documenting.rst [hunk 1]
```diff
@@ -1670,14 +1668,14 @@ in production, other are work in progress:
 .. _github_id: https://github.com/python/python-docs-id
 .. _github_ja: https://github.com/python/python-docs-ja
 .. _github_ko: https://github.com/python/python-docs-ko
-.. _github_pl: https://github.com/python/python-docs-pl
```
Comment (Mariatta) [inline documenting.rst:1673]: This link looks correct.

File: documenting.rst [hunk 2]
```diff
@@ -1658,7 +1657,6 @@ in production, other are work in progress:
 .. _chat_pt_br: https://t.me/pybr_i18n
 .. _doc_ja: https://docs.python.org/ja/
 .. _doc_ko: https://docs.python.org/ko/
-.. _doc_pl: https://docs.python.org/pl/
```
Comment (Mariatta) [inline documenting.rst:1661]: This link is correct.

File: documenting.rst [hunk 3]
```diff
@@ -1792,10 +1790,9 @@ Ask on doc-sig, or better, make a PR on the `devguide
 I have a translation, but not on git, what should I do?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Just ask for help on the doc-sig mailing list and our python-fu, git-fu
-and bash-fu combined will help you create an appropriate repository. If
-you use a tool like transifex don’t worry.  Keeping them in sync is not
-that hard.
+Ask for help on the doc-sig mailing list
```
Comment (Mariatta) [inline documenting.rst:1793]: This section has been updated in #649, so this is good now.

**Review by Mariatta — CHANGES_REQUESTED** [review]
Thanks for the PR. Can you please rebase your branch?

I think most of the links should stay. The only change here is the addition of the `_mail_pl` link. The rest of the change should be reverted.

**Comment by hugovk** [comment]
> Thanks for the PR. Can you please rebase your branch?
> 
> I think most of the links should stay. The only change here is the addition of the `_mail_pl` link. The rest of the change should be reverted.

Please see https://github.com/python/devguide/pull/788 which only adds the `_mail_pl` link.

**Comment by hugovk** [comment]
Closing in favour of https://github.com/python/devguide/pull/788.

**Pull request closed without merging**